src/config/config.cc

```diff
@@ -127,6 +127,9 @@ Config::Config() {
       {"replica-announce-ip", false, new StringField(&replica_announce_ip, "")},
       {"replica-announce-port", false, new UInt32Field(&replica_announce_port, 0, 0, PORT_LIMIT)},
       {"compaction-checker-range", false, new StringField(&compaction_checker_range_, "")},
+      {"force-compact-file-age", false, new Int64Field(&force_compact_file_age, 2 * 24 * 3600, 60, INT64_MAX)},
+      {"force-compact-file-min-deleted-percentage", false,
+       new Int64Field(&force_compact_file_min_deleted_percentage, 10, 1, 100)},
```

Comment:
Why don't you use `IntField` like the one for `profiling-sample-ratio`? The range of values is just from 1 to 100.

Reply:
Yes, `int` type should suffice.